ledger/src/shredder.rs

```diff
@@ -23,6 +23,27 @@ lazy_static! {
         .unwrap();
 }
 
+#[cfg(not(debug_assertions))]
+const ERASURE_BATCH_SIZE: [usize; 33] = [
+    0, 18, 20, 22, 23, 25, 27, 28, 30, // 8
+    32, 33, 35, 36, 38, 39, 41, 42, // 16
+    43, 45, 46, 48, 49, 51, 52, 53, // 24
+    55, 56, 58, 59, 60, 62, 63, 64, // 32
+];
```

Comment:
A comment explaining how these numbers were chosen would be helpful

Reply:
Added some comments. Effectively
```rust
ERASURE_BATCH_SIZE[19] == 46
```
means that if I have only 19 data shreds I want an erasure batch size of 46 (so 27 coding shreds), because a 19:27 erasure batch has the same recovery probabilities as a 32:32 batch.
I used the code below to find out these batch sizes:
https://gist.github.com/behzadnouri/b9c7133f4a4d36394d318ebcacf3c734

Reply:
should this code be checked into the repository?

Reply:
yeah, once these things are finalized, I will update docs and include that calculation.

Reply:
ok, could you file a tracking bug to document this?

Reply:
tracking issue for updating docs: https://github.com/solana-labs/solana/issues/27086